Modules/Video/Core/wrapping/itkVideoSource.wrap

```diff
@@ -0,0 +1,10 @@
+itk_wrap_include("itkImage.h")
+itk_wrap_include("itkVideoStream.h")
+
+itk_wrap_class("itk::VideoSource" POINTER)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
```

Comment:
Perhaps only wrap for 3D images?

Reply:
Can there be a videostream of 3D images?  1D images?

Reply:
@thewtex can correct me here, but yes, my understanding is that `itk::VideoStream` can be templated with any `itk::Image` frame type. Relying on `ITK_WRAP_IMAGE_DIMS` allows the user to create `itk::VideoStream` objects with frames of 2+ dimensions.

With that said I can revisit the constraint on 2D wrapping, it would be useful to allow wrappings for `VideoStream`s of 1D images if the user has included 1D image wrappings.

Reply:
Usually (in my recent PyTorch/MONAI/Nibabel) I've seen ( @thewtex can correct me or @aylward ) folks using 1D, 2D, 3D, and then 4D+channel or 1D scalar, 2D scalar, 2D+Time scalar or 3D scalar (both as 3D), 3D+Time (4D) scalar, and then the same things (1D, 2D, 3D, 4D) as vector images for channel data.  So I'd assume wrapping 1D 2D 3D in this (since you are explicitly handling Time so no 4D) and supporting vector images.

Reply:
Looking back at this point, is there a good way to handle the case where the pixel vector dimension represents time? Probably not the most common case, but that scenario is part of the original motivation for adding this filter. This case is unsupported in the current filter implementation which only handles image axes.

[`VectorImageToImageAdaptor`](https://itk.org/Doxygen/html/classitk_1_1VectorImageToImageAdaptor.html) could be used for handling vector images, but I'm wondering if there is also an efficient mechanism for converting an image of same-length vectors (i.e. `itk::Image<itk::Vector<t,d>,d2>`) to scalar image representation.

Reply:
Going to bump vector image support to a subsequent PR and resolve this thread for now. In theory `VectorImage` frames are already covered as a subset of `Image` frames (other than the edge case where time is represented along vector length), but formalizing support will require additional wrapping for `itk::VideoStream` and a new test + test data.